lib/matplotlib/figure.py

```diff
@@ -1038,6 +1038,42 @@ def fixlist(args):
         key = fixlist(args), fixitems(kwargs.items())
         return key
 
+    def add_artist(self, artist, clip=False):
+        """
+        Add any :class:`~matplotlib.artist.Artist` to the figure.
+
+        Usually artists are added to axes objects using
+        :meth:`matplotlib.axes.Axes.add_artist`, but use this method in the
+        rare cases that adding directly to the figure is necessary.
+
+        Parameters
+        ----------
+        artist : `~matplotlib.artist.Artist`
+            The artist to add to the figure. If the added artist has no
+            transform set to it previously, its transform will be set to
+            ``figure.transFigure``.
+        clip : bool, optional, default ``False``
+            An optional parameter ``clip`` determines whether the added artist
+            should be clipped by the figure patch. Default is *False*,
+            i.e. no clipping.
+
+        Returns
+        -------
+        artist : The added `~matplotlib.artist.Artist`
+        """
+        artist.set_figure(self)
+        self.artists.append(artist)
+        artist._remove_method = self.artists.remove
+
+        if not artist.is_transform_set():
+            artist.set_transform(self.transFigure)
+
+        if clip:
+            artist.set_clip_path(self.patch)
```

Comment:
This should always be passed, unless you can guarantee that the clip path was always `False`, or it should default to `None` if you really don't want to change the existing setting. That is, assuming it has an effect (re. @efiring's comment).

Reply:
Actually, I think I misread this, and thought `clip` was being passed down. Since it isn't, this doesn't really matter.